### PR TITLE
add class CL_0011031 (monocycte-derived dendritic cell)

### LINKED ISSUE #1104: [NTR]monocyte-derived_dendritic_cell
Note - please check that the term does not already exist (check OLS: https://www.ebi.ac.uk/ols/ontologies/cl)

**Preferred term label:**
monocyte-derived dendritic cell

**Synonyms**
MDDC cell

**Definition (free text, please give PubMed ID in format PMID:XXXXXX)**

**Parent cell type term (check the hierarchy here https://www.ebi.ac.uk/ols/ontologies/cl):**

**What is the anatomical structure that the cell is a part of? Please check Uberon: https://www.ebi.ac.uk/ols/ontologies/uberon**
BTO:0002900

**Your nano-attribution (ORCID)**

**Any additional notes or concerns**

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #1104

### CODE REVIEW

**Comment by paolaroncaglia** [comment]
Hi @nicolevasilevsky ,
There are typos in the def "A dendritic cell subset that develops from monocytes monocytes upon stimulation in the circulation that are involved in inflammation and infection."
The BTO xref - should it be a definition dbxref instead? I'm not 100% sure I recall what the resolution was on that type of mappings (Brenda is deprecated/not maintained or not freely accessible, I believe)
CL_0000451 is both in equivalence and subclass, should be removed from latter
Ideally @addiehl may want to review new immune cell terms, but he may not have time at present.

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -23391,6 +23392,16 @@ SubClassOf(obo:CL_0011029 obo:CL_0000151)
 SubClassOf(obo:CL_0011029 ObjectSomeValuesFrom(obo:BFO_0000051 obo:GO_0042151))
 SubClassOf(obo:CL_0011029 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon_6073))
 
+# Class: obo:CL_0011031 (monocycte-derived dendritic cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32733468") obo:IAO_0000115 obo:CL_0011031 "A dendritic cell subset that develops from monocytes monocytes upon stimulation in the circulation that are involved in inflammation and infection.")
+AnnotationAssertion(oboInOwl:created_by obo:CL_0011031 <http://orcid.org/0000-0001-5208-3432>)
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:23398]: Def has typos and is hard to follow. 
What is circulating? Presumably these cells are not involved in infection but it fighting it?
Does this work?
"A dendritic cell that develops from a monocyte.  These cells are involved in inflammation in response to infection"

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -23391,6 +23392,16 @@ SubClassOf(obo:CL_0011029 obo:CL_0000151)
...
+AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0011031 "BTO:0002900")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0011031 "MDDC cell")
+AnnotationAssertion(rdfs:label obo:CL_0011031 "monocycte-derived dendritic cell")
+EquivalentClasses(obo:CL_0011031 ObjectIntersectionOf(obo:CL_0000451 ObjectSomeValuesFrom(obo:RO_0002202 obo:CL_0000576)))
+SubClassOf(obo:CL_0011031 obo:CL_0000451)
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:23403]: Is the subClassOf assertion or can it be inferred?

**Review by dosumis — CHANGES_REQUESTED** [review]

**Comment by dosumis** [comment]
@addiehl  - can you review biology here?

**Comment by nicolevasilevsky** [comment]
@paolaroncaglia thanks for this: https://github.com/obophenotype/cell-ontology/blob/master/docs/Fixing_xsdstring_diffs.md.

I cleaned this up here. :)

File: src/ontology/cl-edit.owl (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/cl-edit.owl:23398]: Sure, I'll update this, then merge it. thanks!

File: src/ontology/cl-edit.owl (hunk 2 above)
Comment (addiehl) [reply src/ontology/cl-edit.owl]: We have always put 'develops from' assertions as subclass axioms, not equivalence axioms.

File: src/ontology/cl-edit.owl (hunk 1 above)
Comment (addiehl) [reply src/ontology/cl-edit.owl:23398]: Also the label has a typo: 'monocycte-derived dendritic cell'  monocyte is spelled wrong

Comment (nicolevasilevsky) [reply src/ontology/cl-edit.owl:23398]: fixed!

File: src/ontology/cl-edit.owl (hunk 2 above)
Comment (nicolevasilevsky) [reply src/ontology/cl-edit.owl]: in addition?

Comment (nicolevasilevsky) [reply src/ontology/cl-edit.owl]: if we don't have the equiv axiom, the parent 'dendritic cell' and 'myeloid leukocyte' is not inferred

**Comment by addiehl** [comment]
Based on the existing subclass axioms for 'myeloid dendritic cell' (CL:0000782), 'develops from' some monocyte, and Nicole's comments above, I think this term should not be added as a separate class at all, but rather as an exact synonyms for 'myeloid dendritic cell'.

**Comment by addiehl** [comment]
The early papers (cited for 'myeloid dendritic cell' CL:0000782) suggest all myeloid dendritic cells are derived from monocytes.  Later sources suggest there may be additional precursors for myeloid DC (PMID:30719026).  Thus, I think we can add 'monocyte-derived dendritic cell' with the asserted parent 'myeloid dendritic cell' and the subclass axiom 'develops from' some monocyte.  We should simultaneously drop the 'develops from ' some monocyte subclass axiom from 'myeloid dendritic cell' since there seem to be additional precursors for this cell type.  Monocytes are themselves myeloid derived, so a DC derived from a monocyte is therefore a type of myeloid dendritic cell (as asserted in the earlier literature).

Sorry it has taken me so long to really look at this issue.

**Comment by addiehl** [comment]
Might be nice to include PMID:30719026 as a dbxref for monocyte-derived dendritic cell

**Comment by nicolevasilevsky** [comment]
Thank you @addiehl! i made all of your suggested changes. Could you (or @dosumis or @paolaroncaglia) please review and approve?

**Review by addiehl — COMMENTED** [review]
Everything looked good

File: src/ontology/cl-edit.owl (hunk 2 above)
Comment (dosumis) [reply src/ontology/cl-edit.owl]: Given the name, it totally makes sense for develops_from to be in the equivalence axiom.  No need to repeat in subClassOf

File: src/ontology/cl-edit.owl (hunk 1 above)
Comment (cmungall) [reply src/ontology/cl-edit.owl:23398]: the label and text def scream out design pattern
